src/components/Form/Field/Field.scss

```diff
@@ -254,7 +254,6 @@
           text-align: center;
           background: $eapp-blue-light;
           border-radius: 50%;
-          padding-right: 0.1rem;
         }
 
         em {
```

Comment:
☝️ Fixing more spacing issues with error messages 😅

Reply:
Just double-checking, but do these CSS changes work well for all 3 cases?
* when there is title, message, and note
* when there is title, message
* when there is just title

Reply:
Yeah the looks good 👍 

![screen shot 2018-07-13 at 8 59 34 am](https://user-images.githubusercontent.com/1449852/42692860-1e33c20a-867b-11e8-9c2c-185f0d2770d3.png)
![screen shot 2018-07-13 at 8 56 36 am](https://user-images.githubusercontent.com/1449852/42692858-1df1d656-867b-11e8-80eb-42c938611b7f.png)
![screen shot 2018-07-13 at 8 56 44 am](https://user-images.githubusercontent.com/1449852/42692859-1e1b46e4-867b-11e8-9367-03ab013ff943.png)